src/main/webapp/js/instructorCourseEdit.es6

```diff
@@ -469,8 +469,26 @@ function editFormRequest(e) {
     });
 }
 
+function enableOnclicks() {
+    $(document).on('click', '.addSectionLevelForInstructor', function () {
+        const index = $(this).attr('data-index');
+        const firstBlankSectionRowIndex = $(this).attr('data-first-blank-section-row-index');
+        showTuneSectionPermissionsDiv(index, firstBlankSectionRowIndex);
+    });
+
+    $(document).on('click', '.toggleSessionLevelInSection', function () {
+        const instructorIndex = $(this).attr('data-instructor-index');
+        const panelIndex = $(this).attr('data-panel-index');
+        const isSectionSpecial = $(this).attr('data-is-section-special') === 'true';
+        $(this).attr('data-is-section-special', isSectionSpecial ? 'false' : 'true');
+        const handler = isSectionSpecial ? hideTuneSessionnPermissionsDiv : showTuneSessionnPermissionsDiv;
+        handler(instructorIndex, panelIndex);
+    });
+}
+
 $(document).ready(() => {
     prepareInstructorPages();
+    enableOnclicks();
```

Comment:
I don't think you should add a function of this name because there are many other `$(document).on('click', ...)`s and `$(...).on('click', ...)`s out there, and it will make people question "why are these click functions special?" while in fact there is nothing special about them.